src/main.cpp

```diff
@@ -304,9 +312,14 @@ void print_help_message() {
   print_option("--output-format=[FORMAT]",
                "Format to print feedback where FORMAT is one of:");
   print_option("", "gnu-like (default if omitted), vim-qflist-json");
+  print_option("--v, --version", "Print version information");
```

Comment:
Shouldn't it be `-v`, not `--v`, for the short form? (See `-h` and `--help`.)

Actually, it looks like `-h` is broken. O_o Maybe we should make `-v` and `-h` work in a separate commit.